docs/core/linux-prerequisites.md

```diff
@@ -215,13 +252,18 @@ To install .NET Core on Debian 8 or 9 (64 bit):
 ## Install .NET Core for Fedora 24, 25, or 26 (64 bit)
 
 > [!Warning]
-> Before you start, remove any previous versions of .NET Core from your system.
+> Before you start, remove any **previous preview** versions of .NET Core from your system.
```

Comment:
I'd make this into a step. But if you don't want to do that, just remove the duplicate one 😄